code/__HELPERS/time.dm

```diff
@@ -66,28 +66,15 @@ GLOBAL_VAR_INIT(rollovercheck_last_timeofday, 0)
 
 
 ///Returns the first day of the current month in number format, from 1 (monday) - 7 (sunday).
-/proc/first_day_of_month()
-	var/year = text2num(time2text(world.timeofday, "YY"))
-	var/month = text2num(time2text(world.timeofday, "MM"))
-
-	var/day = (1 + (((13 * month) + (13 * 1)) / 5) + (year) + (year/4) + (20/4) + (5 * 20)) % 7 //Zeller's congruence
-
-	switch(day) //convert to 1-7 monday first format
-		if(0)
-			day = 6
-		if(1)
-			day = 7
-		if(2)
-			day = 1
-		if(3)
-			day = 2
-		if(4)
-			day = 3
-		if(5)
-			day = 4
-		if(6)
-			day = 5
-	return day
+/proc/first_day_of_month(year, month)
```

Comment:
Document. I know you didn't make it, but you remade it.

Reply:
already is documented